CryptoSwift.xcodeproj/project.pbxproj

```diff
@@ -664,9 +666,11 @@
 				INSTALL_PATH = "$(LOCAL_LIBRARY_DIR)/Frameworks";
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks @loader_path/Frameworks";
 				LLVM_LTO = YES;
+				PRODUCT_BUNDLE_IDENTIFIER = "com.hakore.${PRODUCT_NAME:rfc1034identifier}";
 				PRODUCT_NAME = "$(TARGET_NAME)";
 				SKIP_INSTALL = YES;
-				SWIFT_OPTIMIZATION_LEVEL = "-Ounchecked";
+				SWIFT_DISABLE_SAFETY_CHECKS = YES;
+				SWIFT_OPTIMIZATION_LEVEL = "-O";
```

Comment:
Do you think
SWIFT_DISABLE_SAFETY_CHECKS = YES;			SWIFT_OPTIMIZATION_LEVEL = "-O";
Replace

SWIFT_OPTIMIZATION_LEVEL = "-Ounchecked";